scripts/mender-qemu

```diff
@@ -37,12 +37,16 @@ else
     BUILDTMP=${BUILDDIR}/tmp-glibc
 fi
 
+UBOOT_ELF=${UBOOT_ELF:="${BUILDTMP}/deploy/images/vexpress-qemu/u-boot.elf"}
+VEXPRESS_SDIMG=${VEXPRESS_SDIMG:="${BUILDTMP}/deploy/images/vexpress-qemu/${IMAGE_NAME}-vexpress-qemu.sdimg"}
+QEMU_SYSTEM_ARM=${QEMU_SYSTEM_ARM:="${BUILDTMP}/sysroots/x86_64-linux/usr/bin/qemu-system-arm"}
```

Comment:
Basically if you had written:

```
MY_VAR=${UBOOT_ELF:="${BUILDTMP}/deploy/images/vexpress-qemu/u-boot.elf"}
```

the string would be assigned to both `MY_VAR` *and* `UBOOT_ELF`, which isn't what you would normally expect.